### PR TITLE
Add docker implementation

### PR DESCRIPTION
**Pull request by vrunoa**
Add Dockerfile for creating a Docker container for runsauce

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM node:10
+EXPOSE 8888
```
Comment (jlipps) [inline Dockerfile]: why do we need to expose anything?

File: runsauce.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (jlipps) [inline runsauce.sh]: why is this file necessary?

File: Dockerfile (hunk 1 above)
Comment (vrunoa) [reply Dockerfile]: Copy-paste

File: runsauce.sh (hunk 2 above)
Comment (vrunoa) [reply runsauce.sh]: This wasnt working when I was using CMD, changed to ENTRYPOINT and rollback to use runsauce.js

**Comment by jlipps** [comment]
published as runsauce 2.2.0
